### PR TITLE
Use lerna - reorganise directories accordingly

### PR DESCRIPTION
**Pull request by VictorGosse**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -172,7 +172,18 @@ This project follows all Airbnb React/JSX conventions
 - Always use single quotes, even in JSX.
 - Define propTypes and defaultProps as static property in your component class.
 
-### Files architecture
+## Lerna
+
+This project is using [lerna](https://lerna.js.org/) as it includes multiple packages. You can find them under the /pacakges folder.
```
Comment (alexandre-lelain) [inline CONTRIBUTING.md]: packages ? :P

File: package.json [hunk 2]
```diff
@@ -109,19 +87,13 @@
...
     "webpack": "^4.34.0"
   },
-  "peerDependencies": {
-    "date-fns": "1.30.1",
-    "react": "^16.4.1",
-    "react-dom": "^16.4.1",
-    "styled-components": "^4.0.0"
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged && yarn run lint"
```
Comment (alexandre-lelain) [inline package.json:92]: linter is going to make the pre-commit hook a lot longer, are we sure we want to force this for each commit ?

File: packages/atti-components/rollup.config.js [hunk 3]
```diff
@@ -40,7 +40,7 @@ const getPlugins = (minify = true) => [
     'process.env.NODE_ENV': JSON.stringify(NODE_ENV),
   }),
   babel({
-    exclude: 'node_modules/**',
+    exclude: ['node_modules/**', '../../node_modules/**'],
```
Comment (alexandre-lelain) [inline packages/atti-components/rollup.config.js]: Just a thought here... does it work if you use a regex like `exclude: \/node_modules\/` ? So you don't have to use relative paths

File: packages/website/package.json [hunk 4]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "gatsby-starter-default",
+  "private": true,
+  "description": "A simple starter to get up and developing quickly with Gatsby",
+  "version": "0.1.0",
+  "author": "Kyle Mathews <mathews.kyle@gmail.com>",
```
Comment (alexandre-lelain) [inline packages/website/package.json]: Maybe initialize the file with atti-components meta data ?

File: packages/website/gatsby-config.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
+const path = require("path")
+
+module.exports = {
+  siteMetadata: {
+    title: `Gatsby Default Starter`,
```
Comment (alexandre-lelain) [inline packages/website/gatsby-config.js]: Maybe initialize the file with atti-components meta data ?

File: CONTRIBUTING.md (hunk 1 above)
Comment (VictorGosse) [reply CONTRIBUTING.md]: Fixed

File: packages/website/package.json (hunk 4 above)
Comment (VictorGosse) [reply packages/website/package.json]: Done

File: packages/website/gatsby-config.js (hunk 5 above)
Comment (VictorGosse) [reply packages/website/gatsby-config.js]: Done

File: package.json (hunk 2 above)
Comment (VictorGosse) [reply package.json:92]: It was already like this, I didn't change anything

File: packages/atti-components/rollup.config.js (hunk 3 above)
Comment (VictorGosse) [reply packages/atti-components/rollup.config.js]: Seems to work :)

**Review by totocap — APPROVED** [review]

**Pull request merged**